src/utils/getPreparedTodos.ts

```diff
@@ -0,0 +1,27 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export function getPreparedTodos(
+  todoItems: Todo[],
+  filterStatus: Status,
+  query: string,
+): Todo[] {
+  let filtered = todoItems.filter(todo => {
+    switch (filterStatus) {
+      case Status.All:
+        return true;
```

Comment:
As it was mentioned before you don't need to filter todos if filterStatus is All.